internal/pkg/providers/aws/aws.go

```diff
@@ -24,43 +20,40 @@ type Aws struct {
 	KeepRuleDescriptionPrefix string
 }
 
-var vpcFilter = "vpc-id"
-var groupFilter = "group-name"
-
 // GetName Returns name of provider
 func (a *Aws) GetName() string {
 	return "Amazon Web Services"
 }
 
-// Init initializes the Aws Provider Configuration like Access Token and Reion
+// Init initializes the Aws Provider Configuration like Access Token and Region
 func (a *Aws) Init(params map[interface{}]interface{}) error {
 	err := mapstructure.Decode(params, &a) //Converts the params to Aws struct fields
 	if err != nil {
 		return err
 	}
 	if a.RoleArn == "" || a.Region == "" {
-		return errors.New("Missing Aws Assume Role ARN or Region")
+		return errors.New("missing Aws Assume Role ARN or Region")
 	}
 	return nil
 }
 
 // WhiteListIps - Get List of IP addresses to whitelist
-func (a *Aws) WhiteListIps(resourceIds []string, ipPermissions []utils.IpPermission) error {
+func (a *Aws) WhiteListIpsByLoadBalancer(resourceIds []string, ipPermissions []utils.IpPermission) error {
```

Comment:
The two whitelisting methods are exactly the same except one line. I would suggest to refactor the code and pass filter type in this method so that code can be re-used.